docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,177 @@
+
+```bash
+kyma alpha provision gke --credentials {SERVICE_ACCOUNT_KEY_FILE_PATH} --name {CLUSTER_NAME} --project {GCP_PROJECT} 
+```
+
+To provision a Minikube cluster, run:
+<!---provision minikube is not among the alpha commands--->
+```bash
+kyma alpha provision minikube
+```
+
+## Deploy Kyma
+
+To deploy Kyma using your own domain, run:
+
+```bash
+kyma alpha deploy --domain {DOMAIN} --tls-cert {TLS_CERT} --tls-key {TLS_KEY}
+```
+
+`{TLS_CERT}` and `{TLS_KEY}` are the TLS certificate and TLS key for the domain used. Note that both of them must be encoded in base64. You can use [OpenSSL](https://www.openssl.org/) (which is installed by default with macOS) to convert your TLS certificate and TLS key to base64-encoded values.
+
+You can simplify the process to deploy Kyma with your own domainin  `{DOMAIN}` the following way:
+
+1. Encode the TLS certificate and TLS key, and export them as environment variables:
+  
+    ```bash
+    export TLS_CERT="$(openssl base64 -in {TLS_CERT_FILE_PATH})"
+    export TLS_KEY="$(openssl base64 -in {TLS_KEY_FILE_PATH})"
+    ```
+
+    `{TLS_CERT_FILE_PATH}` is the path to the file containing the TLS certificate, and `{TLS_KEY_FILE_PATH}` is the path to the file containing the TLS key.
+
+2. Use these environment variables in the command:
+
+    ```bash
+    kyma alpha deploy --domain {DOMAIN} --tls-cert $TLS_CERT --tls-key $TLS_KEY
+    ```
+
+To deploy Kyma from the `master` branch, run:
+
+```bash
+kyma alpha deploy --source master
+```
+
+To deploy Kyma using your own Kyma installer image, run:
+
+```bash
+kyma alpha deploy --source {IMAGE}
+```
+
+To build an image from your local sources and deploy Kyma on a remote cluster based on this image, run:
+
+```bash
+kyma alpha deploy --source local --custom-image {IMAGE}
+```
+>    **EXAMPLE:** If your `{IMAGE}` is `user/my-kyma-installer:v1.4.0`, this command builds an image from your local sources and push it your repository `user/my-kyma-installer` with the tag `v1.4.0`.<br>
+Then, this image `user/my-kyma-installer:v1.4.0` is used in the `Deployment` for `kyma-installer`.
+
+To deploy kyma with only specific components, run:
+
+```bash
+kyma alpha deploy --components {COMPONENTS_FILE_PATH}
+```
+`{COMPONENTS_FILE_PATH}` is the path to a YAML file containing the desired component list to be installed, for example:
+```bash
+  components:
+  - name: "cluster-essentials"
+    namespace: "kyma-system"
+  - name: "testing"
+    namespace: "kyma-system"
+  - name: "istio"
+    namespace: "istio-system"
+  - name: "xip-patch"
+    namespace: "kyma-installer"
+  - name: "istio-kyma-patch"
+    namespace: "istio-system"
+  - name: "dex"
+    namespace: "kyma-system"
+  ```
+In this example, only these 6 components are deployed on the cluster.
+
+To override the standard Kyma installation, run:
+<!---didn't find override among the deploy options--->
+```bash
+kyma alpha deploy --override {OVERRIDE_FILE_PATH}
```

Comment:
That changed to the --values approach, overrides is the old concept. Please adjust to it. Remember that we now support providing values by file and by arguments, see the inline help :)

Reply:
We have not two options to provide overrides:

1. As file by setting `--values-file file1.yaml --values-file file2.yaml` or in short-form as`-f file1.yaml...`
1. As single value by setting `--value comp.key=value --value comp.key2=value` (here we don't offer a short-form)

Reply:
That's good to know! I was wondering there the --values come in. 
Do we still use the word "override" in prose text? As in:

To override the standard Kyma installation, you can use the `--values` flag in the following ways:
- Override a file by setting `--values-file file1.yaml --values-file file2.yaml` (short form: `-f file1.yaml...`)
- Override a single value by setting `--value comp.key=value --value comp.key2=value` (no short form available)